src/drivers/drv_led.h

```diff
@@ -49,14 +49,14 @@
 #define BOARD_MAX_LEDS 4
 #endif
 
+#if BOARD_MAX_LEDS < LED_CONTROL_ORB_QUEUE_LENGTH
```

Comment:
This checks looks like it's the wrong way round. And I can't find `LED_CONTROL_ORB_QUEUE_LENGTH` anywhere in this diff. Did you mean `led_control_s::ORB_QUEUE_LENGTH`?

Reply:
Fixed.

Reply:
The ORB queue length must be >= the number of lights on a board. It's cleared in the updated commit.